parquet-pig/src/main/java/org/apache/parquet/pig/summary/SummaryData.java

```diff
@@ -22,26 +22,24 @@
 import java.io.StringReader;
 import java.io.StringWriter;
 
+import com.fasterxml.jackson.core.JsonGenerationException;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
 import org.apache.pig.impl.logicalLayer.FrontendException;
 import org.apache.pig.impl.logicalLayer.schema.Schema;
 import org.apache.pig.impl.logicalLayer.schema.Schema.FieldSchema;
-import org.codehaus.jackson.JsonGenerationException;
-import org.codehaus.jackson.JsonParseException;
-import org.codehaus.jackson.annotate.JsonWriteNullProperties;
-import org.codehaus.jackson.map.JsonMappingException;
-import org.codehaus.jackson.map.ObjectMapper;
-import org.codehaus.jackson.map.SerializationConfig.Feature;
 
 /**
  * Base class for a node of the data summary tree
  */
-@JsonWriteNullProperties(value = false)
```

Comment:
I am not a jackson expert but it seems to me that the default behavior is to write null values (see [here](https://fasterxml.github.io/jackson-core/javadoc/1.9/org/codehaus/jackson/map/annotate/JsonSerialize.html#include())) so removing this annotation might change the expected one.

Reply:
Good point, I've restored the behaviour using the `@JsonInclude(JsonInclude.Include.NON_NULL)` annotation.